README.md

```diff
@@ -17,7 +17,6 @@ This repository includes files with plain SQL that can be used to recreate a dat
   - [🛠 Built With](#built-with)
     - [Tech Stack](#tech-stack)
     - [Key Features](#key-features)
-  - [🚀 Live Demo](#live-demo)
 - [💻 Getting Started](#getting-started)
   - [Setup](#setup)
   - [Prerequisites](#prerequisites)
```

Comment:
![image](https://github.com/ab-noori/vet-clinic/assets/78251927/d1b28b7f-48ca-4071-a5e9-75874eb4fc9e)

**_OPTIONAL_**
- [ ] Please add the missing contents highlighted above for a professional readme 

- [ ] A README file is an essential guide that gives other developers a detailed description of your GitHub project 😊.

- [ ]  It can also be described as documentation with guidelines on how to use a project. Usually, it will have instructions on how to install and run the project 💪

Reply:
**Hi @abdulhamiid 👋,**
Thank you so much 🙏  for reviewing and approving my project.
Your nice feedback is appreciated 🙏, I will consider and make them fix 🎯.


![thank-you-gif-22](https://github.com/ab-noori/bookstore-cms/assets/111179533/0848000e-52ce-440c-a114-6c7f859a0eaa)